addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1905,6 +1904,37 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.Accordion = options.Class.extend({
+    /**
+     * Toggle the position of the cards' collapse state icons
+     * (either floats left (default) or right)
+     *
+     * @see this.selectClass for parameters
+     */
+    iconPos: function (previewMode, widgetValue, params) {
+        const $cards = this.$('.accordion .card');
+        if (widgetValue == "right") {
+            $cards.find('.card-header').addClass('card-header-right');
+        }
+        if (widgetValue == "left") {
+            $cards.find('.card-header-right').removeClass('card-header-right');
+        }
```

Comment:
These 6 lines can be written in one line using `toggleClass` but in fact, you don't even need that entire function at all: same as for style switching, this is only about toggling a class so you can use the `data-select-class` option's method combined with the `data-apply-to` parameter.